packages/core/src/common/metrics.ts

```diff
@@ -27,9 +27,9 @@ export class MetricsService {
   ponder_realtime_latest_block_timestamp: prometheus.Gauge<"network">;
   ponder_realtime_reorg_total: prometheus.Counter<"network">;
 
-  ponder_indexing_total_seconds: prometheus.Gauge<"event">;
-  ponder_indexing_completed_seconds: prometheus.Gauge<"event">;
-  ponder_indexing_completed_events: prometheus.Gauge<"network" | "event">;
+  ponder_indexing_total_seconds: prometheus.Gauge;
+  ponder_indexing_completed_seconds: prometheus.Gauge;
+  ponder_indexing_completed_events: prometheus.Gauge;
```

Comment:
I think we should figure out a way to get indexing function-specific event counts back into the terminal UI.